src/editor/commands/add_positionable_command.cpp

```diff
@@ -1,15 +1,16 @@
 #ifdef TOOLS_ENABLED
 
+#include <modules/isometric_maps/src/editor/isometric_editor_plugin.h>
 #include "add_positionable_command.h"
 
 using namespace editor::commands;
 
 void AddPositionableCommand::redo() {
-    map->add_positionable_if_nothing_present(aabb, positionable_id);
+    IsometricEditorPlugin::get_instance()->get_selected_map()->add_positionable_if_nothing_present(aabb, positionable_id);
```

Comment:
So we move the map out of the command but what if the selected map between different command was changed because someone is editing multiple maps at once and switch between them ?

Reply:
As switch between map is included in `EditorNode`'s undo redo, this should always get the right map.  
The problem of storing map in command is when a user delete a map and then undo this delete. We will have same map data, but not same pointer.